app/styles/_color.scss

```diff
@@ -97,6 +97,8 @@ body {
   --black: #{$black};
 
   --body-type-color: #171717;
+  --am-error-body-text: #121212;
```

Comment:
should check whether `--text-primary` can be visually there is no difference

Reply:
They are different colors.